lib/demos/material/button_demo.dart

```diff
@@ -150,12 +150,55 @@ class _OutlinedButtonDemo extends StatelessWidget {
 // BEGIN buttonDemoToggle
 
 class _ToggleButtonsDemo extends StatefulWidget {
+  _ToggleButtonsDemo({
+    @required this.restorationId,
+  });
+
+  final String restorationId;
+
   @override
   _ToggleButtonsDemoState createState() => _ToggleButtonsDemoState();
 }
 
-class _ToggleButtonsDemoState extends State<_ToggleButtonsDemo> {
-  final isSelected = <bool>[false, false, false];
+class _RestorableBoolList extends RestorableValue<List<bool>> {
```

Comment:
@goderbauer

Reply:
This only works if a new list is assigned to the property every time.

As an alternative, could _ToggleButtonsDemoState just have a List<RestorableBool> with three instances of RestorableBool in it? That may be simpler...